### DISCUSSION TITLE
MongoDB blocked Russian users

### DISCUSSION THREAD

**Discussion by Aziz87** [mongodb/mongoid#5198] [General]
MongoDB - in fascist companies! Distribution of people by national recognition:

**Comment by midnight-wonderer** [comment]
The exact same story with Huawei; sometimes it is not the company; remember, some US companies lose big too. It's the big boss.  
MongoDB is one of the most generous US companies regarding their products, and that's why we love the product, don't you? They are still letting you run the open-source version, no?

If WW3 breaks out, I doubt even Google will keep the promise about their privacy policy.
In wartime, anything happens.

No hard feelings, just my opinions.
My country won't stand a chance in wars against either US or Russia.

**Reply by guy-roberts** [reply]
Bucha
